gradle.properties

```
@@ -0,0 +1,13 @@
+org.gradle.caching=true
+group=io.ballerina.stdlib
+version=0.1.0-SNAPSHOT
+
+ballerinaLangVersion=2201.0.4
```

This should be 2201.4.0
```suggestion
ballerinaLangVersion=2201.4.0
```

---

Addressed in https://github.com/ballerina-platform/module-ballerina-math.vector/pull/1/commits/1fe0f828ef1ea29a4fc27924af5f541af69729ff.